package-lock.json

```diff
@@ -5542,7 +5542,7 @@
 		},
 		"doctrine": {
 			"version": "1.5.0",
-			"resolved": "https://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
+			"resolved": "http://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
```

Comment:
Check if npm resolved these URLs from `https` to `http` before commit `package-lock.json`.